politeiawww/user/user.go

```diff
@@ -497,6 +529,22 @@ type Database interface {
 	// Execute a plugin command
 	PluginExec(PluginCommand) (*PluginCommandReply, error)
 
+	// EmailHistoriesGet returns a map of histories for recipients
+	// [user ID] -> email history (for 24h in the past).
+	//
+	// If the user has no previous history, the result will still contain
+	// a mapping for this user, with existing data (e.g. his email).
+	EmailHistoriesGet(recipients []uuid.UUID) (map[uuid.UUID]EmailHistory, error)
+	// EmailHistoriesSave updates histories map in the database, creating it
```

Comment:
Match the style of the repo you're contributing too.  Do you see other methods without spaces in between them?  Then why are you doing it?

Reply:
I just don't pay attention to opinionated matters like this one, it is not worth it to spend a lot of time on things like that if it is not enforced by the tooling (because every developer has his own style he is comfortable with) - `gofmt`, linters, ...